app/scripts/app.js

```diff
@@ -49,12 +49,7 @@ angular
             rooms : ['$stateParams', 'roomService', 'soundCloudService', '$q', function($stateParams, roomService, soundCloudService, $q) {
               return roomService.fetchRooms($stateParams.mood).then(function(rooms) {
                 return $q.all(rooms.map(function(room) {
-                  return soundCloudService
-                    .fetchMetadata(room.song.soundCloudId)
-                    .then(function(data) {
-                      room.data = data;
-                      return room;
-                    });
+                  return room;
```

Comment:
There is no reason to map anymore. Just return `roomService.fetchRooms($stateParams.mood)`